### PR TITLE
ovn-ic-ecmp refactor

### PR DESCRIPTION
**Pull request by changluyi**
# Pull Request

- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

## What type of this PR

Examples of user facing changes:

- Features
- Bug fixes
- Docs
- Tests

## Which issue(s) this PR fixes

Fixes #(issue-number)

## WHAT

copilot:summary

copilot:poem

## HOW

copilot:walkthrough

### CODE REVIEW

File: pkg/ovn_ic_controller/config.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	OvnTimeout   int
+
+	NodeSwitch     string
+	ClusterRouter  string
+	NodeSwitchCIDR string
+
+	ClusterTCPLoadBalancer        string
+	ClusterUDPLoadBalancer        string
+	ClusterTCPSessionLoadBalancer string
+	ClusterUDPSessionLoadBalancer string
```
Comment (oilbeater) [inline pkg/ovn_ic_controller/config.go:36]: These fields are not used

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Review by oilbeater — APPROVED** [review]

File: pkg/ovn_ic_controller/config.go (hunk 1 above)
Comment (changluyi) [reply pkg/ovn_ic_controller/config.go]: 1

File: pkg/ovs/ovn-ic-sbctl.go [hunk 2]
```diff
@@ -115,6 +123,15 @@ func (c LegacyClient) DestroyRoutes(uuids []string) {
 	}
 }
 
+func (c LegacyClient) DestroyPortBindings(uuids []string) {
+	for _, uuid := range uuids {
+		if err := c.DestroyTableWithUUID(uuid, "Port_Binding"); err != nil {
```
Comment (oilbeater) [inline pkg/ovs/ovn-ic-sbctl.go:128]: should return err when failed

File: pkg/ovn_ic_controller/ovn_ic_controller.go [hunk 3]
```diff
@@ -571,27 +674,24 @@ func (c *Controller) listRemoteLogicalSwitchPortAddress() (*strset.Set, error) {
 	return existAddress, nil
 }
 
-func (c *Controller) getTSName(index int) string {
-	if index == 0 {
-		return util.InterconnectionSwitch
+func moveElements(arr []string, order int) []string {
```
Comment (oilbeater) [inline pkg/ovn_ic_controller/ovn_ic_controller.go:677]:
```suggestion
func removeElement(arr []string, order int) []string {
```

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: pkg/ovs/ovn-ic-sbctl.go (hunk 2 above)
Comment (changluyi) [reply pkg/ovs/ovn-ic-sbctl.go:128]: 1

File: pkg/ovn_ic_controller/ovn_ic_controller.go (hunk 3 above)
Comment (changluyi) [reply pkg/ovn_ic_controller/ovn_ic_controller.go]: it renamed it to generateNewOrdergwNodes

**Review by oilbeater — APPROVED** [review]

**Pull request merged**